common/sgx_workload/workload/ext_work_order_info_kme.h

```diff
@@ -0,0 +1,130 @@
+/* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <stdlib.h>
+
+#include "types.h"
+#include "ext_work_order_info.h"
+
+class ExtWorkOrderInfoKME: virtual public ExtWorkOrderInfo {
```

Comment:
Did you mean to inherit from ExtWorkOrderinfoImpl ? If not, then where will the following functions  be defined ?
GetWorkOrderExData 
VerifyAttestation 
GetWorkorderSigningInfo

Reply:
I didn't intend to inherit ExtWorkOrderInfoKME class from  ExtWorkOrderinfoImpl, rather extend the base interface which is ExtWorkOrderinfo (as ExtWorkOrderInfoKME) and provide implementation (ExtWorkOrderInfoKMEImpl).